### ISSUE TITLE
Changing the style of the @p command

### ISSUE THREAD

**Issue by parmi93** [jothepro/doxygen-awesome-css#162]
I think that readability could be improved by introducing a different style for parameters (`@p`) and code (`@c`).

For example, I made a small style change for the `@p` command with the following result:

Dark mode:
![image](https://github.com/user-attachments/assets/0861a871-fb47-4606-a308-9cfa6ce7560d)

Light mode:
![image](https://github.com/user-attachments/assets/2dbc85c7-1ac7-434c-b845-85efc4ae29ef)

Code documentation:
![image](https://github.com/user-attachments/assets/ff08217e-00a0-4af1-9cba-5203291b4496)

I added the following parameter in the `ALIASES` setting:
`p{1}=<code class="param">\1</code>`

and `custom.css` in the `HTML_EXTRA_STYLESHEET` setting:
```css
code.param {
	font-weight: 600;
	color: var(--primary-dark-color);
}
```

Basically I'm copying the style of the parameter names in the function header, by doing this I can visually distinguish more easily when I am referring to a parameter or when I am referring to simple code.

What I don't like about my solution is that I have to use the `@p{paramName}` syntax instead of `@p paramName`, which makes comments longer and slightly harder to read.

Do you think it would be possible to implement this change directly in `doxygen-awesome-css`? or is there a way I can continue to use the `@p paramName` syntax?

**Comment by jothepro** [comment]
Unfortunately it seems like both `@p` and `@c` are just rendering to `<code>` blocks in HTML. I don't think that there is a way to distinguish them based on a CSS rule.

Please consider opening a PR at [doxygen/doxygen](https://github.com/doxygen/doxygen) for this feature.
